### PR TITLE
Swaager - add support multi package with golang way myPackage.struct

### PR DESCRIPTION
**Pull request by sergeylanzman**
before 
```
// @Success 200 {object} models.mypackage.mystruct desc
```
after
``` 
// @Success 200 {object} mypackage.mystruct desc
```
and now we can have subpackage in models forlder or other folder(not vendor)

### CODE REVIEW

**Comment by astaxie** [comment]
It looks will broken the current comments.

**Comment by sergeylanzman** [comment]
Only if you have models.mypackage.mystruct. If you have one models this work.
Nonetheless today swagger not support with subpackage. 
You get in swagger 
```
&{mypackage mystruct}
```

**Comment by sergeylanzman** [comment]
@astaxie  What do you think about it?

**Comment by astaxie** [comment]
Reviewing, I am still testing in my local with your change.

**Comment by sergeylanzman** [comment]
Thank you

File: g_docs.go [hunk 1]
```diff
@@ -50,13 +52,66 @@ var importlist map[string]string
...
 	controllerList = make(map[string]map[string]*swagger.Item)
 	modelsList = make(map[string]map[string]swagger.Schema)
+	curPath, _ := os.Getwd()
+	astPkgs = map[string]*ast.Package{}
+	parsePackagesFromDir(curPath)
+}
+
+func parsePackagesFromDir(path string) {
+	parsePackageFromDir(path)
+	list, _ := ioutil.ReadDir(path)
```
Comment (astaxie) [inline g_docs.go]: I think need to catch the error here

Comment (sergeylanzman) [inline g_docs.go]: fix

**Comment by sergeylanzman** [comment]
Thank you! :)

**Pull request merged**